app/Foundation/Providers/RouteServiceProvider.php

```diff
@@ -68,8 +68,14 @@ protected function registerBindings()
     public function map(Router $router)
     {
         $router->group(['namespace' => $this->namespace], function (Router $router) {
-            foreach (glob(app_path('Http//Routes').'/*.php') as $file) {
-                $this->app->make('CachetHQ\\Cachet\\Http\\Routes\\'.basename($file, '.php'))->map($router);
+            $path = app_path('Http/Routes');
+
+            foreach (glob("{$path}/*{,/*}.php", GLOB_BRACE) as $file) {
```

Comment:
Yeah, I was unsure where to go on this. So I went trusty `GLOB_BRACE` for one/two depth routes. Would probably be nicer going finder but w/e.